platform/ui/src/components/Select/Select.jsx

```diff
@@ -1,61 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import ReactSelect from 'react-select';
+import ReactSelect, { components } from 'react-select';
 
 import './Select.css';
 
+const MultiValueContainer = ({ data }) => {
+  return <span>{`${data.label}, `}</span>;
```

Comment:
Is it possible to have a comma separating the values only if there are two or more values?
To avoid this:

![image](https://user-images.githubusercontent.com/1905961/77666214-71338700-6f5f-11ea-8103-275d86bc5f2c.png)

Reply:
Array.split(', ') works well for this; then just display the string

Reply:
I will try to work with that. 
Select allows me to override some of its internal components, I was overriding only the MultiSelectValue which only allows me to touch 1 by 1 of the selected items. 

I will work on this to avoid displaying the coma on the last item.

Reply:
Fixed.